common/src/main/java/tachyon/Constants.java

```diff
@@ -197,6 +197,7 @@
   public static final String WORKER_PRINCIPAL_KEY = "tachyon.worker.principal";
   public static final String WORKER_USER_TEMP_RELATIVE_FOLDER = "users";
   public static final String WORKER_DATA_SERVER = "tachyon.worker.data.server.class";
+  public static final String WORKER_COLLECT_METRICS = "tachyon.worker.collect.metrics";
```

Comment:
It doesn't make much sense to just disable client side metrics collection and not worker side. Perhaps we can do something in `BlockMetricsReporter` after the logic is added to that class.